docs/examples/voice.md

```diff
@@ -14,30 +15,79 @@ permalink: examples/voice
 * [Play DTMF Tones into a Call](#play-dtmf-tones-into-a-call)
 * [Error Handling](#error-handling)
 
-## List all Calls
+## Make a Phone Call
 
-A list of all the calls associated with your account.
+Start a call (the from number should be a Nexmo number you own), supplying either `AnswerUrl` *or* `Ncco`:
```

Comment:
I know this has been copied from another part of the file. But should `Nexmo` be `Vonage`?

```suggestion
Start a call (the from number should be a Vonage number you own), supplying either `AnswerUrl` *or* `Ncco`:
```